packages/kit/src/module/install.ts

```diff
@@ -29,7 +29,7 @@ export async function installModule (moduleToInstall: string | NuxtModule, _inli
 
   nuxt.options._installedModules = nuxt.options._installedModules || []
   nuxt.options._installedModules.push({
-    meta: await nuxtModule.getMeta?.(),
+    meta: defu((await nuxtModule.getMeta?.()) || {}, buildTimeModuleMeta || {}),
```

Comment:
`defu` should handle null/undefined/falsy values:

```suggestion
    meta: defu(await nuxtModule.getMeta?.(), buildTimeModuleMeta),
```